css/icons.scss

```diff
@@ -13,6 +13,10 @@
   background-image: url('../img/add-white.svg');
 }
 
+.icon-attach {
+  background-image: url('../img/attach.svg');
```

Comment:
You also need to add an entry to the list here: https://github.com/nextcloud/deck/pull/1305/files#diff-eccb3df06bdb0d0e156bfd782b26d599R58 

That way we can be sure that when using the dark mode the icon is properly inverted.